Exporter/src/main/java/uk/ac/ebi/biostd/exporter/jobs/stats/process/StatsService.java

```diff
@@ -11,18 +11,16 @@
 @AllArgsConstructor
 @Service
 public class StatsService {
-
-    private final MetricsDao metricsDao;
     private final StatsProperties startProperties;
 
     SubStats processSubmission(Submission submission) {
         String accNo = submission.getAccno();
 
         return SubStats.builder()
                 .accNo(accNo)
-                .imaging(submission.isLibFileSubmission())
-                .files(metricsDao.getSubFilesCount(accNo))
-                .filesSize(metricsDao.getSubFilesSize(accNo))
+                .imaging(submission.isImagingSubmission())
+                .files((int) submission.getFilesCount())
```

Comment:
I do not get it why property was declared as long and not int in Submission

Reply:
You're right. Fixed